modules/apps/friendly-url/friendly-url-test/src/testIntegration/java/com/liferay/friendly/url/service/test/FriendlyURLEntryLocalServiceTest.java

```diff
@@ -65,6 +73,52 @@ public void tearDown() {
 			_classNameLocalService.getClassNameId(User.class));
 	}
 
+
+	@Test
+	public void testAddFriendlyURLEntryWithCategories()
+		throws Exception {
+
+		ServiceContext serviceContext = _getServiceContext());
```

Comment:
There's a compile error here (extra bracket at the end). As a general rule, commits should never include errors of this kind; can you squash the fix into this commit? This kind of things makes reviews more difficult.